habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -482,31 +482,56 @@ def _load_navmesh(self, ep_info: RearrangeEpisode):
 
         navmesh_path = osp.join(base_dir, "navmeshes", scene_name + ".navmesh")
 
-        if osp.exists(navmesh_path):
-            self.pathfinder.load_nav_mesh(navmesh_path)
-            logger.info(f"Loaded navmesh from {navmesh_path}")
+        # if osp.exists(navmesh_path):
+        #     self.pathfinder.load_nav_mesh(navmesh_path)
+        #     logger.info(f"Loaded navmesh from {navmesh_path}")
+        # else:
+        #     logger.warning(
+        #         f"Requested navmesh to load from {navmesh_path} does not exist. Recomputing from configured values and caching."
+        #     )
```

Comment:
If we intend to preserve caching, I would toggle it via a config field.

If not, I would either add a comment above this block or remove it entirely.